support-frontend/assets/helpers/user/userReducer.js

```diff
@@ -48,28 +45,13 @@ const initialState: User = {
 
 // ----- Functions ----- //
 
-function defaultStateOrProvince(countryGroup: CountryGroupId): string {
```

Comment:
There are no circumstances where we would want to default a state to the first in the list, either the user must select their correct state (helped by the pre-selection from Geo-IP or their existing account), or it isn't required and it's sent through blank or null.